fast_api_als/routers/lead_conversion.py

```diff
@@ -37,34 +39,43 @@ def get_quicksight_data(lead_uuid, item):
         "3pl": item.get('3pl', 'unknown'),
         "oem_responded": 1
     }
+    info("accepting Lead info pulled from DDB"
+    info("lead_hash " +data[lead_hash] + " model " + data[model] + " postalcode " +postalcode + " dealer " + dealer)
     return data, f"{item['make']}/1_{int(time.time())}_{lead_uuid}"
 
 
 @router.post("/conversion")
 async def submit(file: Request, token: str = Depends(get_token)):
+    start_time = time.time()
     body = await file.body()
     body = json.loads(str(body, 'utf-8'))
 
     if 'lead_uuid' not in body or 'converted' not in body:
-        # throw proper HTTPException
+            # throw proper HTTPException
+        exception("lead_uuid or converted doesn't exist in submited file")
```

Comment:
Not a good practice to throw any type of Exception.
throw proper HTTPException.